content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -205,10 +205,8 @@ spec:
 ```
 
 Service account bearer tokens are perfectly valid to use outside the cluster and
-
 can be used to create identities for long standing jobs that wish to talk to the
-Kubernetes API. To manually create a service account, simply use the `kubectl`
-
+Kubernetes API. To manually create a service account, simply use the `kubectl
```

Comment:
To manually create a service account, ~~simply~~ use the

Reply:
May I ask why it's better to avoid the word 'simple'?

Reply:
> May I ask why it's better to avoid the word 'simple'?


See https://github.com/kubernetes/website/issues/17070 for (IMO) a good explanation on this.